plugins/msr/fu-plugin-msr.c

```diff
@@ -41,12 +53,23 @@ gboolean
 fu_plugin_startup (FuPlugin *plugin, GError **error)
 {
 	FuPluginData *priv = fu_plugin_get_data (plugin);
+	guint eax = 0;
 	guint ecx = 0;
 
 	/* sdbg is supported: https://en.wikipedia.org/wiki/CPUID */
-	if (!fu_common_cpuid (0x01, NULL, NULL, &ecx, NULL, error))
-		return FALSE;
-	priv->ia32_debug_supported = ((ecx >> 11) & 0x1) > 0;
+	if (fu_common_get_cpu_vendor () != FU_CPU_VENDOR_INTEL) {
```

Comment:
wait, don't you want to be checking **for** Intel?

Reply:
Indeed, doh.